includes/Story_Post_Type.php

```diff
@@ -524,12 +655,14 @@ public function get_editor_settings() {
 					'media'       => '/web-stories/v1/media',
 					'link'        => '/web-stories/v1/link',
 					'statusCheck' => '/web-stories/v1/status-check',
+					'metaboxes'   => $meta_box_url,
 				],
 				'metadata'         => [
 					'publisher' => $this->get_publisher_data(),
 				],
 				'version'          => WEBSTORIES_VERSION,
 				'encodeMarkup'     => $this->decoder->supports_decoding(),
+				'metaBoxes'        => $this->get_meta_boxes_per_location(),
```

Comment:
I don't think we should add this unless this is feature flag is enabled. We could do this 
```
if ( $this->experiments->is_experiment_enabled( 'customMetaBoxes' ) ) {
     $settings['config']['metaBoxes'] = $this->get_meta_boxes_per_location();
     // Get admin url for handling meta boxes.
     $meta_box_url = admin_url( 'post.php' );
     $meta_box_url = add_query_arg(
          [
              'post'                  => $story_id,
              'action'                => 'edit',
              'meta-box-loader'       => true,
              'meta-box-loader-nonce' => wp_create_nonce( 'meta-box-loader' ),
          ],
          $meta_box_url
     );
     $settings['api']['metaboxes'] = $meta_box_url;
}
```

Reply:
Seems unnecessary. In the editor it's a no-op anyway if the feature is disabled.

Reply:
Why run the methods when they are not used?

Reply:
It also hide the feature completely unless the flag is enabled.